### DISCUSSION TITLE
Simulated Plant Chicken Problem [Mengchuan Wu, Zheng Chen]

### DISCUSSION THREAD

**Discussion by mengchuanWu** [realitydeslab/playshop-summer-2024#16] [Proposal]
ChineseTitle：仿真植物 鸡的问题
Research Question：
The reverse domestication of human culture by simulated plant technology
仿真植物技术对人类文化的反向驯化

Keywords：
Technology itself invisibly shapes humanity，Reinvention of culture，Science ethics，Emulate，Plant Diversity
技术本身无形的塑造人类，文化的再塑造，科技伦理，仿真，植物多样性

Artefact / Media:
Video
影像

Abstract：
Based on the current flourishing development of artificial plants and the advancement of related technologies (such as plant algorithm gene models and biological 3D printing technology), we predict that artificial plants will extensively cover human living areas in the future.

Using the development of chickens as a reference for the future development of artificial plants, we speculate that in the future, artificial plants might alter the structural distribution of "plants" (the encroachment of artificial plants on the living space of real plants might be similar to the current situation where artificially bred chickens squeeze out other species of chickens). They might even replace real plants and reshape our understanding of what constitutes a plant (artificial plants could become our entire understanding of the definition of plants, similar to how our current perception of chickens is completely based on the appearance of artificially bred chickens).

We discuss and draw parallels between this issue and the issue of chickens to help people understand the potential social problems that artificial plants might bring in the future. This is not just about the problem with chickens or the potential issue with artificial plants; they are all part of the objective social problems that arise from the unrestricted use of human technology.

基于当下仿真植物的发展繁荣现状和相关技术发展（植物算法基因模型和生物3D打印技术），我们预测未来仿真植物会大范围覆盖人类生活领域。

以鸡的发展经历作为仿真植物的未来发展参照，我们推测未来仿真植物可能会改变“植物”的结构分布（未来仿真植物对真实植物生存空间的侵占类似于当下的人工培育的鸡对其它种类鸡的生存空间挤压），甚至替代真实植物重塑我们对于植物的认知（仿真植物未来可能成为我们对植物这一定义的全部理解，就像现在我们认知中的鸡完全是人工饲养鸡的种类的样貌）。

我们将这一问题与鸡的问题一起讨论和类比以便人们理解仿真植物在未来可能带来的社会问题，这不仅仅是鸡的问题或是可能出现的仿真植物的问题，他们都是人为技术无限制使用下所造成的客观社会问题的一部分呈现而已。

References:
1.Green 3D bioprinting of plant cells: A new scope for 3D bioprinting
植物细胞的绿色3D生物打印：3D生物打印的新领域
2.Evolved Control of Natural Plants: Crossing the Reality Gap for User-Defined Steering of Growth and Motion
对天然植物的进化控制：跨越现实差距，实现用户定义的生长和运动转向
3.Simulation Modeling of Plants and Plant
模拟建模植物和植物生态系统
4.Generating Plants with Gene Expression Programming
使用基因表达编程生成植物
5.Research on High-Efficiency Simulation Technology for Complex System
复杂系统高效仿真技术研究
6.3D Bioprinting to Fabricate Living Microalgal Materials
3D生物打印制造活微藻材料

**Comment by mengchuanWu** [comment]
![微信图片_20240626141843](https://github.com/realitydeslab/playshop-summer-2024/assets/173230949/efd0f8b4-7fc2-4d1b-96d1-42411f500ef1)
![微信图片_20240626141733](https://github.com/realitydeslab/playshop-summer-2024/assets/173230949/bb52955d-94ea-46c2-8e5c-6108e77a6c99)
![微信图片_20240626141738](https://github.com/realitydeslab/playshop-summer-2024/assets/173230949/b22e0c3c-5546-42ee-a75a-0b2ed33182fc)
![微信图片_20240626141847](https://github.com/realitydeslab/playshop-summer-2024/assets/173230949/65398366-1c56-4481-9c25-801802962374)

**Comment by mengchuanWu** [comment]
![微信图片_20240627131342](https://github.com/realitydeslab/playshop-summer-2024/assets/173230949/170639df-8e53-4dba-a843-af83ec04bc89)

**Comment by mengchuanWu** [comment]
<img width="931" alt="微信图片_20240627135243" src="https://github.com/realitydeslab/playshop-summer-2024/assets/173230949/3c2b06c7-3dc0-47fc-965e-8f69aae975dd">

**Comment by mengchuanWu** [comment]
https://github.com/realitydeslab/playshop-summer-2024/assets/173230949/71410af1-1396-4819-8b7c-12e4a853e394

**Comment by glitch-kraft** [comment]
![1418854](https://github.com/realitydeslab/playshop-summer-2024/assets/46427283/094154ed-1653-4e06-8e70-3e6437438ebf)

**Reply by glitch-kraft** [reply]
Лукоморье

**Comment by glitch-kraft** [comment]
All is full of love – Bjork

https://vimeo.com/43444347

**Comment by glitch-kraft** [comment]
https://pinkchickenproject.com/

Andreas Greiner
MONUMENT FOR A CONTEMPORARY DINOSAUR (ROSS 308)

**Comment by cz-and** [comment]
Tagline:仿真植物，科技重塑自然未来，重新定义植物，改变生活，正如人工鸡改变禽类世界。
Simulated plants: Redefining nature through technology, transforming our understanding of flora and reshaping life, just as artificial chickens have transformed the avian world.

**Comment by cz-and** [comment]
[仿真植物.pdf](https://github.com/user-attachments/files/16061984/default.pdf)

**Comment by mengchuanWu** [comment]
![微信图片_20240702131423](https://github.com/realitydeslab/playshop-summer-2024/assets/173230949/fb8879a9-f8c6-4ee3-b71e-e4e676f2c98b)

**Comment by mengchuanWu** [comment]
Keyframe Description:The assembly line is constantly running, the mechanical arm is working frantically to produce simulated plants, the light and shadow in the woods are constantly changing, the wind is blowing gently, and the leaves are floating down.
关键帧描述:流水线不断的运转,机械臂疯狂的工作生产仿真植物,树林中的光影不断变化,有风轻轻吹过,落叶飘下.

**Comment by glitch-kraft** [comment]
**Title**
Simulated Plant. Chicken Problem

**Tagline**

What will happen to plant diversity if we flood the world with artificial plants, just as we have done to bird diversity by breeding chickens?
